pkg/function/scale_workload.go

```diff
@@ -115,14 +113,27 @@ func (s *scaleWorkloadFunc) ExecutionProgress() (crv1alpha1.PhaseProgress, error
 	}, nil
 }
 
-func getArgs(tp param.TemplateParams, args map[string]interface{}) (namespace, kind, name string, replicas int32, waitForReady bool, err error) {
+type scaleArgs struct {
+	namespace    string
+	kind         string
+	name         string
+	replicas     int32
+	waitForReady bool
+}
+
+func getArgs(tp param.TemplateParams, args map[string]interface{}) (*scaleArgs, error) {
 	var rep interface{}
-	waitForReady = true
-	err = Arg(args, ScaleWorkloadReplicas, &rep)
+	waitForReady := true
+	err := Arg(args, ScaleWorkloadReplicas, &rep)
 	if err != nil {
-		return
+		return nil, err
 	}
 
+	var (
+		namespace, kind, name string
+		replicas              int32
+	)
```

Comment:
```suggestion
		var namespace, kind, name string
		var replicas int32
```
The space between  replicas and int32 looks weird

Reply:
It's auto-formatting, based on `namespace, kind, name string`.

Reply:
My suggestion was to declare `var` on two lines instead of using `var()` pattern

Reply:
Ohh, I misunderstood it somehow. Address the review comment now.